src/log_reader.cpp

```diff
@@ -275,7 +275,7 @@ namespace
         }
         entry["msg"] = d;
         if (!entry.isMember("level")) {
-            std::any_of(LibWbMqttLogLevels.begin(), LibWbMqttLogLevels.end(), [&](const auto& p) {
+            (void)std::any_of(LibWbMqttLogLevels.begin(), LibWbMqttLogLevels.end(), [&](const auto& p) {
```

Comment:
Как-то оно стало странновато выглядеть
```suggestion
            auto it=std::find_if(LibWbMqttLogLevels.begin(), LibWbMqttLogLevels.end(), [&](const auto& p) {
            return StringStartsWith(d, p.first);
            });
            if (it != LibWbMqttLogLevels.end()) {
               entry["level"] = it.second;
            }
```